### PR TITLE
fix: update fmp metric

### PR DESCRIPTION
**Pull request by Fine0830**
Fixes https://github.com/apache/skywalking/issues/8812

Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

**Comment by wu-sheng** [comment]
I think there are 3 other PRs not added into changelogs, https://github.com/apache/skywalking-client-js/milestone/8?closed=1.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Apache SkyWalking Client JS
 - Lightweight
 - Make browser as a start of whole distributed tracing
 
+NOTICE,  SkyWalking Client JS 0.8.0 and later versions use SkyWalking v9.
```
Comment (wu-sheng) [inline README.md:11]:
```suggestion
NOTICE,  SkyWalking Client JS 0.8.0 and later versions require SkyWalking v9.
```

**Review by wu-sheng — APPROVED** [review]

File: src/trace/type.d.ts [hunk 2]
```diff
@@ -28,8 +28,8 @@ export interface SpanFields {
   startTime: number;
   endTime: number;
   spanId: number;
-  spanLayer: number;
-  spanType: number;
+  spanLayer: string;
+  spanType: string;
```
Comment (wu-sheng) [inline src/trace/type.d.ts:32]: Are these working for the HTTP span receiver? FYI @arugal

Comment (Fine0830) [reply src/trace/type.d.ts:32]: > Are these working for the HTTP span receiver? FYI @arugal

Yes, I  have checked the protocol https://github.com/apache/skywalking/blob/master/docs/en/protocols/Trace-Data-Protocol-v3.md. And I found the mistake from https://github.com/apache/skywalking-client-js/pull/76

**Comment by arugal** [comment]
The E2E failed. Please recheck @Fine0830

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Changes
 
+## 0.8.0
+
+1. Fix fmp metric.
+2. Add e2e tese based on skywaling-infra-e2e.
+3. Update metric and events.
+4. Add span layers and types.
+
```
Comment (wu-sheng) [inline CHANGES.md:9]:
```suggestion
5. Remove ServiceTag by following SkyWalking v9 new layer model.
```

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**